lib/src/participant/local.dart

```diff
@@ -304,4 +305,29 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
       }
     }
   }
+
+  /// Control who can subscribe to LocalParticipant's published tracks.
+  ///
+  /// By default, all participants can subscribe. This allows fine-grained control over
+  /// who is able to subscribe at a participant and track level.
+  ///
+  /// Note: if access is given at a track-level (i.e. both [allParticipantsAllowed] and
+  /// [allTracksAllowed] are false), any newer published tracks
+  /// will not grant permissions to any participants and will require a subsequent
+  /// permissions update to allow subscription.
+  ///
+  /// @param allParticipantsAllowed Allows all participants to subscribe all tracks.
```

Comment:
does Dart use the same `@param` syntax in comments as Java?

Reply:
doesn't look like it's recommended  😅 https://dart.dev/guides/language/effective-dart/documentation#do-use-prose-to-explain-parameters-return-values-and-exceptions